pmd-java/src/test/java/net/sourceforge/pmd/lang/java/ast/ParserCornersTest.java

```diff
@@ -128,7 +129,8 @@ public void testLambdaBug1333() {
 
     @Test
     public void testLambdaBug1470() throws Exception {
-        String code = IOUtils.toString(ParserCornersTest.class.getResourceAsStream("LambdaBug1470.java"), "UTF-8");
+        String code = IOUtils.toString(ParserCornersTest.class.getResourceAsStream("LambdaBug1470.java"),
```

Comment:
We should make use of the method `readAsString(...)` instead.

Reply:
Done.